src/features/evaluations/[id]/EvaluationsList.tsx

```diff
@@ -0,0 +1,145 @@
+import { useEffect, useState } from "react"
+import { useParams, useNavigate } from "react-router-dom"
+import { Badge } from "../../../components/badge/Badge"
+import { useAppSelector } from "../../../hooks/useAppSelector"
+import { LinkButton } from "../../../components/button/Button"
+import { Icon } from "../../../components/icon/Icon"
+import { EvaluationStatus } from "../../../types/evaluationType"
+import { getUserEvaluations } from "../../../redux/slices/userSlice"
+import { useAppDispatch } from "../../../hooks/useAppDispatch"
+import { Loading } from "../../../types/loadingType"
+import { getEvaluationStatusVariant } from "../../../utils/variant"
+import { type Evaluation } from "../../../types/evaluationType"
+
+export const EvaluationsList = () => {
+  const navigate = useNavigate()
+  const { id } = useParams()
+  const appDispatch = useAppDispatch()
+  const { loading, user_evaluations } = useAppSelector((state) => state.user)
+  const [sortedEvaluations, setSortedEvaluations] = useState<Evaluation[]>([])
+
+  useEffect(() => {
+    if (id !== undefined) {
+      void getEvaluations()
+    }
+  }, [id])
+
+  const getEvaluations = async () => {
+    if (id !== undefined) {
```

Comment:
move this inside `useEffect` ? 🤔

Reply:
Oki kuys, I followed kasi yung how getTemplates was implemented din sa EvaluatorsRoles.tsx